pallets/elections/src/lib.rs

```diff
@@ -34,6 +36,7 @@ use sp_std::{
 pub type TotalReward = u32;
 
 const STORAGE_VERSION: StorageVersion = StorageVersion::new(4);
+pub(crate) const LOG_TARGET: &str = "pallet-elections";
```

Comment:
Just `pub`, please. Although, why do you want to use this outside of the pallet anyway?

Reply:
rm `pub(crate)`